src/filterScreenings.js

```diff
@@ -0,0 +1,61 @@
+/*Filters out ten screenings in the next five days or removes the last day's screenings incase
+the number of screenings pushes the total number of screenings to more than ten*/
+export function filterScreenings(data) {
+  let filteredScreenings = [];
+  data.forEach((screening) => {
+    
+    // Setting a starting data for the next day to start displaying screenings
+    const currentScreeningDate = new Date(screening.attributes.start_time);
+    const startingPoint = new Date(Date.now() + 86400000);
```

Comment:
I think this is a fine solution technically, but to more easily communicate what's going on to a reader of the code, it's common to write out all the factors: `24 * 60 * 60 * 1000`. Most readers will then be able to understand that we're talking about 24 hours worth of milliseconds, which is much more difficult to dduce from `86400000`.